sklearn/ensemble/stacking.py

```diff
@@ -0,0 +1,265 @@
+"""Stacked ensembles"""
+
+# Author: Caio Oliveira <caioaao@gmail.com>
+# License: BSD 3 clause
+
+from ..base import (BaseEstimator, TransformerMixin, MetaEstimatorMixin, clone)
+from ..model_selection import cross_val_predict
+from ..pipeline import FeatureUnion
+from ..preprocessing import FunctionTransformer
+
+
+class StackingTransformer(BaseEstimator, MetaEstimatorMixin, TransformerMixin):
+    """Transformer to turn estimators into meta-estimators for model stacking
```

Comment:
I don't think that mentioning meta-estimators will be useful for user. I recall some comments from @amueller.
 
```
Transformer combining (or stacking) the predictions of several estimators.
```